org.eclipse.m2e.core.tests/resources/projects/demo/.gitignore

```diff
@@ -0,0 +1,33 @@
+HELP.md
```

Comment:
Is git ignore file relevant for the test? if not we can safely remove it from the PR...